contents.json

```diff
@@ -1455,6 +1455,11 @@
       "category": "date",
       "description": "Library for dealing with `NSDate`, `NSCalendar`, `NSDateComponents`, and `NSTimeInterval`.",
       "homepage": "https://github.com/SwiftKitz/Datez"
+    }, {
+      "title": "NVDate",
+      "category": "date",
+      "description": "Swift4 Date extension library.",
```

Comment:
@novalagung could you remove `Swift4` from the description?

Reply:
sure